### DISCUSSION TITLE
Does not have tag search option in UI

### DISCUSSION THREAD

**Discussion by lippoliv** [jonaswinkler/paperless-ng#1631] [Feature Requests]
#401 does talk about searching for tag sets (all/any) which I like. The "not" filter for Tags is mentioned there too.

I would like to see that NOT filter added to the UI? We could work with a three-click solution here:

- 1st click includes the tag to the search
- 2nd click excludes the tag to the search
- 3rd click does not care about that tag at all (which is the default case)

Currently the user just has 1st and 3rd option. 

### Use case

Find all documents with the tag "inbox" which do not have the tag "todo".

### Partially implemented

Using saved views, you can achieve it (complicated).

1. Create a saved view for "inbox" tag
2. Go to admin section
3. Edit the view
4. Add an "does not have tag" option (you can just use IDs here, no autocomplete support)

**Comment by shamoon** [comment]
I implemented this already, unfortunately another PR that lies in waiting… see #399
